### ISSUE TITLE
Update to use latest go-whisk code

### ISSUE THREAD

**Issue by paulcastro** [openwhisk/openwhisk-client-go#1]
This is currently out-of date.  Need to update to latest code in whisk core, add build instructions to make sure wsk18in gets built correctly.

**Comment by dubeejw** [comment]
Why exactly does this repo exist at the current time? Not only is it out of date, the original commits were not copied over from the main repo.

**Comment by dubeejw** [comment]
I used one of the below commands to create the Python client repo (https://github.com/openwhisk/openwhisk-client-python). Notice all of the commits are preserved.

remove folders from history
```
git filter-branch --prune-empty --tree-filter 'rm -rf $(find . -type d | egrep -v "\.$" | egrep -v tools/cli | egrep -v tools | xargs)' -f HEAD
```
remove all files excluding a directory
```
git filter-branch --prune-empty --tree-filter 'rm -rf $(find . -type f -not -path "./tools/cli*”)’ -f HEAD
```

**Comment by paulcastro** [comment]
@dubeejw thanks looks good.

This repo was originally a placeholder.  We have some technical debt to pay here.  Still open issue  to [move CLI and go-whisk out of core](https://github.com/openwhisk/openwhisk/issues/1480)

**Comment by paulcastro** [comment]
Addressed in merge of #160
